contrib/babelfishpg_tsql/src/pl_exec-2.c

```diff
@@ -3817,3 +3830,110 @@ exec_stmt_change_dbowner(PLtsql_execstate *estate, PLtsql_stmt_change_dbowner *s
 	update_db_owner(stmt->new_owner_name, stmt->db_name);	
 	return PLTSQL_RC_OK;
 }
+
+static int
+exec_stmt_createfulltextindex(PLtsql_execstate *estate, PLtsql_stmt_createfulltextindex *stmt)
+{
+	char        *table_name;
+	char		*index_name;
+	bool		index_exists = true;
+	char		*query_str;
+	const char		*schema_name;
+	Oid 		schemaOid;
+	Oid			relid;
+	List		*column_name;
+
+	schema_name = gen_schema_name_for_fulltext_index((char *)stmt->schema_name);
+	schemaOid = LookupExplicitNamespace(schema_name, true);						
+	table_name = stmt->table_name;
+	column_name = stmt->column_name;
+	index_name = stmt->index_name;
+
+	// check if schema exists
+	if (!OidIsValid(schemaOid))
+			ereport(ERROR,
+				(errcode(ERRCODE_UNDEFINED_SCHEMA),
+					errmsg("schema \"%s\" does not exist",
+						stmt->schema_name)));
+
+	relid = get_relname_relid((const char *) table_name, schemaOid);
+
+	// check if table exists
+	if (!OidIsValid(relid))
+			ereport(ERROR,
```

Comment:
generic comment
1) We should have test case for each error we are throwing, in executor or ANTLR
2) We should have test case for each condition if/else we have. For example : creating full text index on one column vs multiple column

Reply:
Yes, I've added those test cases in the JDBC test files of full text index.